.github/workflows/fuzzing.yml

```diff
@@ -2,6 +2,7 @@ name: Fuzzer execution
 
 on:
   workflow_dispatch:
+  pull_request:
```

Comment:
>pull_request: [](http://example.com/codeflow?start=2&length=13)

Most project CI runs complete in seconds. Ours is already very long. Adding 5 minutes for fuzzing makes it longer. 

Why cannot we run the fuzzer just nightly and for longer duration rather than fuzz at every PR?

Reply:
I think nightly and longer fuzzing is probably going to give us more bang for our buck. As i don't know how much benefit the 5 minutes will bring, but i do no the 5 minutes more in a pipelines can be quite annoying to dev's want to build.

Since this is an area of contention and because for the task at hand we only need to run for the release. I would suggest the nightly run and then revisit this at a later date to fully understand the benefit of the PR path.
I would suggest creating a workitem to make sure we follow up at a later date.